src/backend/blockchain/src/routes.py

```diff
@@ -21,9 +50,29 @@ def index():
 def get_chain():
     chain = []
     for block in blockchain.chain:
-        chain.append(block.toJSON())
+        chain.append(block.to_json())
     return jsonify(length=len(chain), chain=chain), HttpCode.OK.value
 
+@app.route('/addPeer', methods=['POST'])
+def add_peer():
+    data = request.get_json()
+    try:
+        peer = data["peer"]
+        peers.append(peer)
+    except KeyError:
+        IncorrectPayloadException()
+
+    return jsonify(success=True), HttpCode.OK.value
```

Comment:
this is not being used currently but might be needed for testing

Reply:
Don't forget to update when you learn whether or not it's needed for testing

Reply:
pretty useful if you are testing locally!